app/assets/javascripts/angular/directives/index-body.js.erb

```diff
@@ -21,7 +21,7 @@ app.directive('onFinishRender', ['$timeout', function ($timeout){
             // isRTL: true,
             // isFitWidth: true
           });
-        });
+        }, 1000);
```

Comment:
this is 500mil sec
![500 milsec screen shot 2015-01-29 at 12 03 05 am](https://cloud.githubusercontent.com/assets/8570304/5959393/d11a2404-a808-11e4-97f9-65f38ba40d89.png)